src/main/java/com/hrznstudio/titanium/Titanium.java

```diff
@@ -84,6 +78,8 @@
     public static final String MODID = "titanium";
     public static final Logger LOGGER = LogManager.getLogger(MODID);
     public static NetworkHandler NETWORK = new NetworkHandler(MODID);
+    public static final PluginManager RESOURCES = new PluginManager(MODID, FeaturePlugin.FeaturePluginType.FEATURE, featurePlugin -> featurePlugin.value().equalsIgnoreCase(ResourceRegistry.PLUGIN_NAME), false);
```

Comment:
If the main idea is to check if their name matches, couldn't there be a somewhat cleaner method? Even if there is a constructor that takes a string and turns it into this?